hooks/pre-push

```diff
@@ -93,16 +93,11 @@ pylint_check "catlearn/" 2#000011
 # Run pylint on the tests
 pylint_check "tests/" 2#000011 "no-self-use,duplicate-code"
 
-
-# Github #23 - https://github.com/arreason/CategoryLearning/issues/23
-# Deactivate mypy checks, since we have a handful of outstanding errors/warnings
```

Comment:
I'd rather not remove this comment, since it explains we have a deactivated mypy check in the pre-push hook. Either add it back or remove altogether `type_check`